public/docs/ts/latest/cookbook/a1-a2-quick-reference.jade

```diff
@@ -584,21 +582,21 @@ table(width="100%")
       code-example.
         &lt;tr ng-repeat="movie in movieList | orderBy : 'title'">
       :marked
-        Orders the collection, as specified by the expression.
+        `OrderBy` orders the collection, as specified by the expression.
```

Comment:
`OrderBy` doesn't seem right—shouldn't it be `orderBy`?

But actually, it seems weird to repeat it here. How about rephrasing like this:

Sorts the collection in the order specified by ... 

(I'm not sure about this, because I see "ordering or sorting" elsewhere, which makes me wonder if there's a difference I'm not getting here. https://en.wikipedia.org/wiki/Sorting says there are two common meanings of "sorting": ordering & categorizing. Hm.)